Cura-5.x-M5-Profile/definitions/ankermake_m5.def.json

```diff
@@ -60,6 +61,7 @@
         "machine_height": { "default_value": 250 },
         "machine_name": { "default_value": "AnkerMake M5" },
         "machine_start_gcode": { "default_value": "M104 S{material_print_temperature_layer_0} ; set final nozzle temp\nM190 S{material_bed_temperature_layer_0} ; set and wait for nozzle temp to stabilize\nM109 S{material_print_temperature_layer_0} ; wait for nozzle temp to stabilize\nG28 ;Home\nG1 E10 F3600; push out retracted filament(fix for over retraction after prime)" },
+        "machine_end_gcode": {"default_value": "M104 S0\nM140 S0\n;Move nozzle 8mm up from print\nG91\nG1 Z8\n;Retract the filament\nG92 E1\nG1 E-1 F300\nG28 X0 Y0\nM84"},
```

Comment:
should we change the max print height to 242 because of this? If you think we should then we should add a not in the change.